lab_1_classify_by_unigrams/main.py

```diff
@@ -64,7 +92,14 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    squa_diff = 0
```

Comment:
the same

Reply:
I can't. Otherwise, Python has nothing to turn to.

Reply:
What? I mean inverse the condition!

Reply:
done